spread/runner.go

```diff
@@ -289,10 +289,7 @@ func (r *Runner) prepareContent() (err error) {
 	}
 	include := r.project.Include
 	if len(include) == 0 {
-		include, err = filterDir(r.project.Path)
```

Comment:
I'm missing the actual motivation of the change. It looks like these are not doing the same thing. Description says it simplifies.. but it doesn't talk about the rationale and consequences for the actual behavior change.